app/javascript/report_form_multiselect.vue

```diff
@@ -0,0 +1,438 @@
+<template lang="pug">
+div
+  multiselect(
+    v-model='selected',
+    :options='options',
+    placeholder='',
+    :multiple='true',
+    label='title',
+    track-by='title',
+    @select='select($event)',
+    @remove='remove($event)'
+  )
+  #multiselect__values
+</template>
+
+<script>
+import Multiselect from 'vue-multiselect'
+export default {
+  components: {
+    Multiselect
+  },
+  props: {
+    practices: { type: String, required: true },
+    editpractices: { type: [Array], default: () => [] }
+  },
+  data() {
+    const jsonPractices = JSON.parse(this.practices)
+    const practices = jsonPractices.map((practice) => {
+      const robj = { title: practice[0], id: practice[1] }
+      return robj
+    })
+    return {
+      selected: [],
+      options: practices,
+      editdata: this.editpractices || []
+    }
+  },
+  computed: {},
+  mounted() {
+    for (const data of this.editdata) {
+      for (const practice of this.options) {
+        if (data === practice.id) {
+          this.selected.push({ title: practice.title, id: practice.id })
```

Comment:
新しくオブジェクトを作る必要がないかも？

```suggestion
          this.selected.push(practice)
```